changelogs/fragments/264-allow-user_medias-to-be-optional.yaml

```
@@ -0,0 +1,2 @@
+minor_changes:
+  - zabbix_user - user_medias now defaults to None and is optional.
```

```suggestion
  - zabbix_user - user_medias now defaults to None and is optional (https://github.com/ansible-collections/community.zabbix/pull/264).
```

Please add this PR URL to the end.